packages/orbit-components/src/Tag/index.jsx

```diff
@@ -120,16 +115,7 @@ export const StyledTag: any = styled.div`
         box-shadow: none;
       }
 
-      &:focus {
-        background: ${getBackgroundColor(STATES.DEFAULT)};
```

Comment:
it should have focus-ring and background. So you will have to get background back ;)

Reply:
I am ok with that, but does it really? I thought that was because hover and focus styles were kinda mixed.
So, if I do that, when we have a tag focused and we hover it, we have no visual feedback that we are hovering it. Should it be like that?
For now, I'll add the background effect back as you requested, but I think we should confirm with designers, if it hasn't been confirmed already.

Reply:
this background color applies after you click on it. So I guess we should not care about hover, because it was already hovered and clicked. If it was focused by Tab, then you have a cursor pointer and it should be enough to let you know that it's hovered ;) Ask designers to be sure on 100% percents, but that how it was designed initially ;)

Reply:
Clarified [here](https://skypicker.slack.com/archives/GSGN9BN6Q/p1666858014346839).